src/battle-animations.ts

```diff
@@ -693,17 +691,33 @@ class BattleScene {
 			if (i % 3 === 2) pokemonhtml += `</div><div class="teamicons">`;
 		}
 		pokemonhtml = '<div class="teamicons">' + pokemonhtml + '</div>';
+		const ratinghtml = side.rating ? ` title="Rating: ${BattleLog.escapeHTML(side.rating)}"` : ``;
+		let posStr = "";
+		if (isAlly) {
+			if (!side.isFar) {
+				posStr += "p3";
+			} else {
+				posStr += "p4";
+			}
+		} else {
+			posStr = !side.isFar ? "p1" : "p2";
+		}
+		return `<div class="trainer" id="${posStr}"><strong>${BattleLog.escapeHTML(side.name)}</strong><div class="trainersprite"${ratinghtml} style="background-image:url(${Dex.resolveAvatar(side.avatar)})"></div>${pokemonhtml}</div>`;
```

Comment:
```suggestion
		return `<div class="trainer trainer-${posStr}"><strong>${BattleLog.escapeHTML(side.name)}</strong><div class="trainersprite"${ratinghtml} style="background-image:url(${Dex.resolveAvatar(side.avatar)})"></div>${pokemonhtml}</div>`;
```

Reply:
See what I said in `battle.css` - IDs should straight-up never be used for any reason in modern code.